src/components/ui/companyProfile/companyDetail/company-contact.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { LocalIcon, IconName } from '@/assets/icons/local-icon';
+import { LinkSocialItemPros } from "@/components/ui/companyProfile/types";
+
+export const LinkSocialItem: React.FC<LinkSocialItemPros> = ({
+  icon,
+  link,
+  title,
+}) => {
+  return (
+
+    // Company Contact Section
+    <a
+      href={link}
+      target="_blank"
+      rel="noopener noreferrer"
+      className="inline-flex items-center px-3 py-2 gap-2 border border-[#D6DDEB] rounded-[4px] text-[#4640DE] text-sm font-medium"
+    >
```

Comment:
```suggestion
    <Link
      to={link}
      target="_blank"
      rel="noopener noreferrer"
      className="inline-flex items-center px-3 py-2 gap-2 border border-[#D6DDEB] rounded-[4px] text-[#4640DE] text-sm font-medium"
    >
```
and config color in tailwind.config.js